ci/CLOUD_BUILD.md

```diff
@@ -0,0 +1,47 @@
+# Installing Spinnaker for GCP on Google Cloud Build
+
+## Shared VPC
+
+Please note that using Cloud Build to install Spinnaker for GCP with a Shared VPC host project is currently unsupported. For Shared VPC support, conduct the [setup](https://cloud.google.com/docs/ci-cd/spinnaker/spinnaker-for-gcp) in Cloud Shell.
+
+## Service account
+
+Google Cloud Build has a default service account that can be found in the settings page of the Cloud Build UI. The Cloud Build service account must be configured with the following roles to install Spinnaker for GCP:
+
+- Cloud Functions Developer - roles/cloudfunctions.developer
```

Comment:
It is a bit tricky because if the user is configuring through the UI they will be see the roles listed as the longform names, if they are adding them through the CLI they will need to add them using the role-paths

Reply:
Makes sense. Let it stand.